src/h5web/visualizations/matrix/MatrixVis.tsx

```diff
@@ -35,7 +35,7 @@ function MatrixVis(props: Props): JSX.Element {
       }
     >
       <div ref={divRef} className={styles.wrapper}>
-        {isVisible && (
+        {isMeasured && (
```

Comment:
Random renaming I thought of.

Reply:
I think I preferred `isVisible`. 

To me, *visible* is the state whereas the *measure* is how we determine it. I therefore find that `isVisible` makes more sense as a condition.